addons/project/static/src/core/common/message_patch.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<template xml:space="preserve">
+    <t t-name="mail.message_template" t-inherit="mail.Message" t-inherit-mode="extension">
+        <xpath expr="//div[contains(@class, 'o-mail-Message-body')]" position="attributes">
+            <attribute name="class" add="invisible" separator=" "/>
+        </xpath>
+    </t>
+</template>
```

Comment:
Can you explain why you did that change? 🤔 

FYI, that changes will impact all `mail.Message`. 😨

Reply:
Hello @xavierbol ,
Indeed, it will impact all mail messages. In this task, we added a notification to the mail template. 
![Uploading chatter notification.png…]()
However, upon making this change, we observed its effect on the lognote. To address this issue, I made the necessary adjustments.

Reply:
Hello @xavierbol ,
Done, I had made changes to address the issue without affecting all mail.Message .